SocialCareCaseViewerApi/V1/Boundary/Requests/CreateAllocationRequest.cs

```diff
@@ -0,0 +1,17 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace SocialCareCaseViewerApi.V1.Boundary.Requests
+{
+    public class CreateAllocationRequest
+    {
+        [Required]
+        [MaxLength(16)]
+        public long MosaicId { get; set; }
+
+        [MaxLength(50)]
+        public string WorkerEmail { get; set; } = null;
```

Comment:
We can't allow null values for WorkerEmail for new allocations, so this should be a required field.